platform/security/servlet/security-servlet-whoami/src/test/groovy/org/codice/ddf/security/servlet/whoami/WhoAmITest.groovy

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.servlet.whoami
+
+import ddf.security.Subject
+import ddf.security.SubjectUtils
+
+class WhoAmITest extends SubjectSpecification {
```

Comment:
❗️ Our [defined standard for Spock tests](https://codice.atlassian.net/wiki/display/DDF/Unit+Testing#UnitTesting-Spock) is to suffix test classes with `Spec` or `Specification`.